UI/src/app/lp-map/lp-map.component.ts

```diff
@@ -40,6 +45,21 @@ export class LpMapComponent implements OnInit {
 
   onMapReady(map: Map): void {
     this.map = map;
+
+    const geojsonMarkerOptions = {
+      radius: 5,
+      fillColor: "#ffff02",
+      color: "#ffff02",
+      weight: 1,
+      opacity: 1,
+      fillOpacity: 0.5
+    };
+
+    this.streetLightsService.getStreetLights(this.map.getBounds().toBBoxString()).subscribe(lampLocation => {
+      this.layers.push(geoJSON<FeatureCollection>(lampLocation, {
+        pointToLayer: (feature, latlng) => (circleMarker(latlng, geojsonMarkerOptions))
+      }));
+    });
```

Comment:
Does this add street lights as you move around the map?

Take a look at the [possible map events](https://github.com/Asymmetrik/ngx-leaflet#mouse-interactions-leafletmouseevent) to make this call when a user drags the map around. You may need to find a way to make sure that duplicate streetlights aren't added in the same view.

Reply:
Resolved!